stable/index.yaml

```diff
@@ -1550,7 +1550,21 @@ entries:
     version: 0.9.0
   - apiVersion: v1
     appVersion: 1.1.0
-    created: "2020-08-11T23:43:26.964519036Z"
+    created: "2020-08-12T21:22:38.151554-07:00"
+    description: Kommander
+    digest: 0f5b9961e909dc1c5a8af103521bb7f808650a64679023ee1c266836dd74333b
+    home: https://github.com/mesosphere/charts
+    maintainers:
+    - name: hectorj2f
+    - name: alejandroEsc
+    - name: jimmidyson
```

Comment:
heh at this point you might want to put your name in it 😉

Reply:
I thought this was supposed to happen automatically when we merge to m/charts? Or does that only happen on master, hence this PR?

Reply:
oh oops, I didn't even realize I wasn't 😬 
the publish job does happen automatically with merge to master, the process is just tricky here because we are patching a previous version of the chart - latest version on master is 0.9.4, but we want to patch 0.8.x